modules/distr-tracing-install-jaeger-operator.adoc

```diff
@@ -5,48 +5,45 @@ This module included in the following assemblies:
-* If you require persistent storage, you must also install the OpenShift Elasticsearch Operator before installing the {JaegerName} Operator.
-
-[WARNING]
-====
-Do not install Community versions of the Operators. Community Operators are not supported.
-====
+* If you require persistent storage, you must install the {es-op} before installing the {JaegerOperator} Operator.
 
 .Procedure
 
 . Log in to the {product-title} web console as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
 
 . Navigate to *Operators* -> *OperatorHub*.
 
-. Type *distributed tracing platform* into the filter to locate the {JaegerName} Operator.
+. Search for the {JaegerOperator} Operator by entering *distributed tracing platform* in the search field.
 
-. Click the *{JaegerName} Operator* provided by Red Hat to display information about the Operator.
+. Select the *{JaegerOperator}* Operator, which is *provided by Red Hat*, to display information about the Operator.
 
 . Click *Install*.
 
-. On the *Install Operator* page, select the *stable* Update Channel. This automatically updates your Operator as new versions are released.
+. For the *Update channel* on the *Install Operator* page, select *stable* to automatically update the Operator when new versions are released.
 //If you select a maintenance channel, for example, *Stable*, you will receive bug fixes and security patches for the length of the support cycle for that version.
 
 . Accept the default *All namespaces on the cluster (default)*. This installs the Operator in the default `openshift-operators` project and makes the Operator available to all projects in the cluster.
 
-* Accept the default *Automatic* approval strategy. By accepting the default, when a new version of this Operator is available, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator without human intervention. If you select *Manual* updates, when a newer version of an Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to have the Operator updated to the new version.
+. Accept the default *Automatic* approval strategy.
 +
 [NOTE]
 ====
-The *Manual* approval strategy requires a user with appropriate credentials to approve the Operator install and subscription process.
+If you accept this default, the Operator Lifecycle Manager (OLM) automatically upgrades the running instance of this Operator when a new version of the Operator becomes available.
+
+If you select *Manual* updates, the OLM creates an update request when a new version of the Operator becomes available. As a cluster administrator, you must then manually approve the update request to have the Operator updated to the new version. The *Manual* approval strategy requires a user with appropriate credentials to approve the Operator installation and subscription process.
```

Comment:
```suggestion
If you select *Manual* updates, the OLM creates an update request when a new version of the Operator becomes available. As a cluster administrator, you must then manually approve the update request to update the Operator to to the new version. The *Manual* approval strategy requires a user with appropriate credentials to approve the Operator installation and subscription process.
```

Reply:
It might be helpful to be more specific as to what the `appropriate credentials` are.

Reply:
I've reworded the sentence in your second comment here. Thank you.